diff --git a/jf-common.mk b/jf-common.mk
index 3a86266..7f355d2 100644
--- a/jf-common.mk
+++ b/jf-common.mk
@@ -187,7 +187,8 @@ PRODUCT_PROPERTY_OVERRIDES += \
     persist.rild.nitz_short_ons_1="" \
     persist.rild.nitz_short_ons_2="" \
     persist.rild.nitz_short_ons_3="" \
-    ro.telephony.ril.v3=newDriverCall
+    ro.telephony.ril.v3=newDriverCall \
+    dalvik.vm.dexopt-data-only=0
 
 # call common msm8960
 $(call inherit-product, device/samsung/msm8960-common/msm8960.mk)
